packages/procaptcha-bundle/src/index.tsx

```diff
@@ -111,8 +111,8 @@ const renderLogic = (
     config: ProcaptchaClientConfigInput,
     renderOptions?: ProcaptchaRenderOptions
 ) => {
-    elements.forEach((element) => {
-        const callbackName = renderOptions?.callback || element.getAttribute('data-callback')
+    elements.forEach((element) => { // || element.getAttribute('data-callback')
```

Comment:
Remove this instead of commenting out please

Reply:
Okay, I got it